client/me/account/main.jsx

```diff
@@ -711,7 +724,9 @@ const Account = createReactClass( {
 	},
 
 	handleSubmitSuccess( response, formName = '' ) {
-		this.props.markSaved && this.props.markSaved();
+		if ( ! this.hasUnsavedUserSettings( [].concat( ACCOUNT_FIELDS, INTERFACE_FIELDS ) ) ) {
+			this.props.markSaved && this.props.markSaved();
```

Comment:
I guess we no longer need the `this.props.markSaved &&` check, or do we?

Reply:
removed in https://github.com/Automattic/wp-calypso/pull/49760/commits/3577e3b701189a8c9f908fbb170e61bc9f26b78a